Sources/LiveKit/Publications/RemoteTrackPublication.swift

```diff
@@ -184,7 +199,15 @@ private extension RemoteTrackPublication {
 
     var userCanModifyTrackSettings: Bool {
         // adaptiveStream must be disabled and must be subscribed
-        !isAdaptiveStreamEnabled && subscribed
+        if kind == .video && isAdaptiveStreamEnabled {
+            log("Adaptive stream is enabled, cannot change video track settings", .warning)
+            return false
+        }
+        if !(preferSubscribed || subscribed) {
+            log("Cannot update track settings when not subscribed", .warning)
+            return false
+        }
+        return true
     }
```

Comment:
@davidzhao Do you mean this whole check is not required anymore ?

Reply:
I just mean the `subscribed` check. `isAdaptiveStreamEnabled` is still needed. though re-reading the code, you now have `preferSubscribed` which actually covers it. sorry about the false alarm.